Front/src/app/services/customer.service.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { CustomerViewModel } from '../models/CustomerViewModel';
+import { Observable } from 'rxjs';
+import { HttpResponseModel } from '../models/HttpResponseModel';
+import { AppConfig } from '../common/app-config';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CustomerService {
+
+  constructor(private http: HttpClient) { }
+  
+  ServiceEndpoint: string = `${AppConfig.API_URL}`;
```

Comment:
Concatenate the controller name here and avoid it in every method.
```suggestion
  ServiceEndpoint: string = `${AppConfig.API_URL}/api/Customer`;
```